apps/wallet/src/ui/app/staking/selectors.ts

```diff
@@ -49,11 +49,9 @@ export function getValidatorSelector(validatorAddress?: string) {
     return createSelector(suiSystemObjectSelector, (systemObj) => {
         const { data } = systemObj || {};
         if (data?.dataType === 'moveObject') {
-            const { active_validators: active, next_epoch_validators: next } =
-                data.fields.validators.fields;
+            const { active_validators: active } = data.fields.validators.fields;
```

Comment:
@Jordan-Mysten after this change, we will need to get next epoch validators differently. We could do this by either:
1. deriving that on the client side
2. exposing an api in fullnode for this 

But the comment in line 48 seems to indicate we are looking for something different?

Reply:
was told by @Jordan-Mysten  offline that he had another change to fetch the validators in a different way